services/web-server/src/login/PostgresSessionStore.js

```diff
@@ -52,6 +52,8 @@ module.exports = function ({ session, db, options = {} }) {
 
         if (!row) {
           return callback();
+        } else if (db.decrypt({ value: row.data.encrypted_session_id }).toString('utf8') !== sessionId){
```

Comment:
I think you want just `row.encrypted_session_id` here -- `data` is the session data itself.  That should fix the errors about `kid` in the tests.